relay/messages/execution.go

```diff
@@ -73,11 +72,11 @@ var errorCommandNotFound = errors.New("Command not found")
 // ToCircuitRequest converts an ExecutionRequest into a circuit.api.ExecRequest
 func (er *ExecutionRequest) ToCircuitRequest(bundle *config.Bundle, relayConfig *config.Config, useDynamicConfig bool) (*api.ExecRequest, bool, error) {
 	retval := &api.ExecRequest{}
-	hasDynamicConfig := er.compileEnvironment(retval, relayConfig, useDynamicConfig)
-	command := bundle.Commands[er.commandName]
+	command := bundle.Commands[er.CommandName()]
```

Comment:
I reordered the call to `compileEnvironment`. Before after `compileEnvironment` was called that attribute would be set. Now I'm just pulling out the command first, moving up the error, and then sending that in to `compileEnvironment` to include extra env vars if needed.